[8.16] Filtering out cold and frozen data tiers during rule execution (#5849)
elastic/security-docs: docs/detections/detections-exclude-cold-frozen-data-tiers.asciidoc

[[exclude-cold-frozen-data-individual-rules]]
== Exclude cold and frozen data from a rule

:frontmatter-description: Configure a rule to ignore cold and frozen data during execution. 
:frontmatter-tags-products: [security]
:frontmatter-tags-content-type: [how-to]
:frontmatter-tags-user-goals: [manage]

> Rules that query cold and frozen data might perform slower and have longer execution times. To exclude cold and frozen data, add a Query DSL filter that forces the rule to ignore cold and frozen {ref}/data-tiers.html[data tiers] when executing. You can add the filter when creating a new rule or updating an existing one. 

Reviewer: Some word choice nits:
- "perform slower" and "have longer execution times" sound like the same thing. Pick one?
- "forces the rule to ignore" sounds a little, well, forceful. Probably could omit without losing meaning.

```suggestion
Rules that query cold and frozen data might perform more slowly. To exclude cold and frozen data, add a Query DSL filter that ignores cold and frozen {ref}/data-tiers.html[data tiers] when executing. You can add the filter when creating a new rule or updating an existing one. 
```
